### PR TITLE
Bug 2096780: Wizard with sysprep in additionalResources

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

same as #831  but for sysprep.

User can edit and detect sysprep config using template objects array

### CODE REVIEW

File: src/views/catalog/wizard/tabs/scripts/components/Sysprep.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  const currentVMSysprepName = currentSysprepVolume?.sysprep?.configMap?.name;
+
+  const filterSysprepByName = isSysprepConfig(currentVMSysprepName);
+
+  const sysPrepObject = tabsData?.additionalObjects?.find(
+    filterSysprepByName,
+  ) as IoK8sApiCoreV1ConfigMap;
+
+  const unattend = sysPrepObject?.data?.[UNATTEND];
+  const autoUnattend = sysPrepObject?.data?.[AUTOUNATTEND];
```
Comment (metalice) [inline src/views/catalog/wizard/tabs/scripts/components/Sysprep.tsx:41]: can be in a single line

Comment (metalice) [inline src/views/catalog/wizard/tabs/scripts/components/Sysprep.tsx:41]: can be in a single line

Comment (upalatucci) [reply src/views/catalog/wizard/tabs/scripts/components/Sysprep.tsx:41]: Okay great

**Pull request merged**